src/patient-registration/patient-relationship-registration.test.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
```

Comment:
This file could be moved to `src/patient-registration/custom-inputs/`. But we can make a separate ticket for this once it has been merged into it's own branch.